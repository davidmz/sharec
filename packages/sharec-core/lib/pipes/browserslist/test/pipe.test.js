const { fixtures } = require('testUtils')
const { pipe } = require('../pipe')

describe('strategies > pipes > browserslist', () => {
  const browserslistBaseFxt = fixtures('browserslist/lines/00-base')

  it('should process browserslist configs', () => {
    expect(pipe('.browserslistrc')(browserslistBaseFxt)).toMatchSnapshot()
  })
})
