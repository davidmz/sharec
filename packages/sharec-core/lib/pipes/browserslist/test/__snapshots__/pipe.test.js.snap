// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`strategies > pipes > browserslist should process browserslist configs 1`] = `
"last 1 version
> 1%
maintained node versions
not dead"
`;
